terraform/api_gateway.tf.json.template.py

```diff
@@ -181,8 +181,8 @@ def waf_match_path(path_regex: str) -> JSON:
 def add_waf_blocked_alarm(resources: JSON) -> JSON:
     """
     Add a metric alarm that trips if the ratio between blocked and overall
-    requests goes above 25%. Note that requests blocked by rules listed in
-    :py:attr:`Config.waf_rules_not_logged` are not considered.
+    requests goes above the set threshold. Note that requests blocked by rules
+    listed in :py:attr:`Config.waf_rules_not_logged` are not considered.
```

Comment:
```suggestion
    requests goes above 25% in prod and 50% elsewhere. Note that
```

Reply:
I think it's a mistake to mention the actual value in the docstring since it causes unnecessary duplication, and can be misleading if the value is ever changed and the docstring is not.